app/commodities/page.client.tsx

```diff
@@ -40,7 +40,9 @@ const PageClient = () => {
 
     // TODO: submit data to backend
     setTimeout(() => {
+      /* eslint-disable */
```

Comment:
remove these so the console logs remain as warnings. They remind us to return to these functions and add substance to them.